library/cwm/examples/object_api.rb

```diff
@@ -47,8 +47,6 @@ class ExampleDialog
     include Yast::I18n
     include Yast::UIShortcuts
     def run
-      textdomain "example"
```

Comment:
The same here, something like `# FIXME: to have translations set a text domain here`